apps/hash-api/src/graph/linear-types.ts

```diff
@@ -0,0 +1,1436 @@
+    await LINEAR_TYPES_INITIALIZERS.linkEntityType.snoozedBy(context);
+
+  const snoozedUntilAtPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.snoozedUntilAt(context);
+
+  const sortOrderPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.sortOrder(context);
+
+  const startedAtPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.startedAt(context);
+
+  const startedTriageAtPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.startedTriageAt(context);
+
+  const subIssueSortOrderPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.subIssueSortOrder(context);
+
+  const subscriberLinkEntityType =
+    await LINEAR_TYPES_INITIALIZERS.linkEntityType.subscriber(context);
+
+  // const teamLinkEntityType =
+  //   await LINEAR_TYPES_INITIALIZERS.linkEntityType.team(context);
+
+  // const teamEntityType = await LINEAR_TYPES_INITIALIZERS.entityType.team(
+  //   context,
+  // );
+
+  const titlePropertyType = await LINEAR_TYPES_INITIALIZERS.propertyType.title(
+    context,
+  );
+
+  const trashedPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.trashed(context);
+
+  const triagedAtPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.triagedAt(context);
+
+  const updatedAtPropertyType =
+    await LINEAR_TYPES_INITIALIZERS.propertyType.updatedAt(context);
+
+  const urlPropertyType = await LINEAR_TYPES_INITIALIZERS.propertyType.url(
+    context,
+  );
+
+  /* eslint-enable @typescript-eslint/no-use-before-define */
+
+  return entityTypeInitializer({
+    ...linearTypes.entityType.issue,
+    properties: [
+      {
+        propertyType: archivedAtPropertyType,
+        required: false,
+      },
+      {
+        propertyType: autoArchivedAtPropertyType,
+        required: false,
+      },
+      {
+        propertyType: autoClosedAtPropertyType,
+        required: false,
+      },
+      {
+        propertyType: branchNamePropertyType,
+        required: true,
+      },
+      {
+        propertyType: canceledAtPropertyType,
+        required: false,
+      },
+      {
+        propertyType: completedAtPropertyType,
+        required: false,
+      },
+      {
+        propertyType: createdAtPropertyType,
+        required: true,
+      },
+      {
+        propertyType: customerTicketCountPropertyType,
+        required: true,
+      },
+      {
+        propertyType: markdownDescriptionPropertyType,
+        required: false,
+      },
+      {
+        propertyType: dueDatePropertyType,
+        required: false,
+      },
+      {
+        propertyType: estimatePropertyType,
+        required: false,
+      },
+      {
+        propertyType: idPropertyType,
+        required: true,
+      },
+      {
+        propertyType: identifierPropertyType,
+        required: true,
+      },
+      {
+        propertyType: numberPropertyType,
+        required: true,
+      },
+      {
+        propertyType: previousIdentifierPropertyType,
```

Comment:
```suggestion
        propertyType: previousIdentifiersPropertyType,
```

Reply:
This is intentional so that the property type is reusable on entity types whether in an array or not

Reply:
Discussed offline, the property types should be singular